.github/workflows/build-in-devcontainer.yaml

```
@@ -55,7 +55,7 @@ jobs:
             echo "HAS_DEVCONTAINER=false" >> "${GITHUB_ENV}";
           fi
       - if: ${{ env.HAS_DEVCONTAINER == 'true' }}
-        uses: aws-actions/configure-aws-credentials@v4
+        uses: aws-actions/configure-aws-credentials@v2
```

Wondering if this should be v3.
```suggestion
        uses: aws-actions/configure-aws-credentials@v3
```